File: codeup/100000572/D.py
#!/usr/bin/python3

class student:
    def __init__(self, num, name, score1, score2, score3):
        self.num = num
        self.name = name
        self.score1 = score1
        self.score2 = score2
        self.score3 = score3


def my_input():
    student_list = []
    for i in range(5):
        num, name, score1, score2, score3 = list(input().split())
        student_list.append(student(num, name, score1, score2, score3))

    return student_list


def my_print(student_list):
    for ent in student_list:
        print(ent.num, ent.name, ent.score1, ent.score2, ent.score3)


student_list = my_input()
my_print(student_list)

File: codeup/100000572/D.cpp
#include <iostream>
using namespace std;

struct student {
  int num;
  char name[20];
  int score1;
  int score2;
  int score3;
};

void my_input(student *list) {
  for (int i = 0; i < 5; i++)
    scanf("%d %s %d %d %d", &list[i].num, list[i].name, &list[i].score1,
          &list[i].score2, &list[i].score3);
  return;
}

void my_print(student *list) {
  for (int i = 0; i < 5; i++)
    printf("%d %s %d %d %d\n", list[i].num, list[i].name, list[i].score1,
           list[i].score2, list[i].score3);
  return;
}
int main() {
  student list[5];
  my_input(list);
  my_print(list);
  return 0;
}